src/web/app/pages-session/session-result-page/session-result-page.component.ts

```diff
@@ -251,21 +255,13 @@ export class SessionResultPageComponent implements OnInit {
           this.courseId,
           this.previewAsPerson,
           this.regKey,
-        ).subscribe((student: Student) => {
+        ).pipe(finalize(() => {
+          this.isPersonLoading = false;
+          this.logStudentAccess();
```

Comment:
Would it be possible that the loading fails and we try to log an invalid student?

Reply:
Yes it is possible and it will show `Failed to log feedback session view` to the user, would it be better to call the function right after the required info is assigned (eg line 264) instead?

https://github.com/TEAMMATES/teammates/blob/ec82613a7bc7d63d7927b0297cd13b26185fa327/src/web/app/pages-session/session-result-page/session-result-page.component.ts#L262-L264

Reply:
Yes, because I think users don't usually know they're being logged, and it might confuse them if an error like that is shown to them